mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/query/parsing/nativequery/impl/NativeQueryParser.java

```diff
@@ -367,6 +369,16 @@ public Rule MapReduce() {
 		);
 	}
 
+	public Rule Drop() {
+		return Sequence(
+				"drop",
```

Comment:
s/"drop"/"drop "

Reply:
Fixed.

I wonder why we need the space though. Anyway, I've updated the code